hs_core/hydroshare/utils.py

```diff
@@ -459,6 +463,8 @@ def copy_resource_files_and_AVUs(src_res_id, dest_res_id, set_to_private=False):
         if value:
             if avu_name == 'isPublic' and set_to_private:
                 istorage.setAVU(dest_coll, avu_name, 'False')
+            elif avu_name == 'quotaUserName' and requesting_user:
+                istorage.setAVU(dest_coll, avu_name, requesting_user.username)
```

Comment:
OK, I am confused. 

Looks like you're changing the quota attribution for a resource when files are being copied into it. This does not seem to be correct. The AVU assignment should be made at resource creation time, not at the time that a file is copied into the thing. Otherwise there will be significant churn in resource ownership...?